packages/core/src/components/tooltip/tooltip.scss

```diff
@@ -11,12 +11,6 @@
   visibility: collapse !important;
   overflow-wrap: break-word;
 
-  border-radius: 0.25rem;
```

Comment:
`border-radius` should remain on host element otherwise you can get this kind of effect (red corners):

![image](https://github.com/user-attachments/assets/5a792402-72a3-4e45-afbe-2f53468c2955)